suite-native/app/globalPolyfills.js

```diff
@@ -14,36 +17,6 @@ install();
 
 // The Buffer implementation from react-native-quick-crypto is not compatible with Trezor Connect.
 global.Buffer = require('buffer').Buffer;
-// There is bug in buffer polyfill that when you call subarray it returns Uint8Array instead of Buffer, this fixes it
```

Comment:
stellar-sdk internally uses `Buffer`, and patching it here will not modify the `Buffer` used within stellar-sdk, so I have introduced [`@exodus/patch-broken-hermes-typed-arrays`](https://github.com/ExodusMovement/patch-broken-hermes-typed-arrays), which is used by well-known wallets like Rabby Wallet and Exodus Wallet. This may require evaluation by your team; if you think it's unnecessary, I can also apply the patch internally in stellar-sdk.